### ISSUE TITLE
Can't build static pages

### ISSUE THREAD

**Issue by moebiusmania** [empress/ember-ghost#13]
Hi, I was having trouble building static pages, so I've tried to re-create a project from scratch using the stepts suggested on readme:

```
$ npm install -g ember-cli
$ ember new super-blog
$ cd super-blog
$ ember install ember-ghost ember-ghost-casper-template
$ ember build -e production
```

but I get these errors (multiple times):

```
Error while processing route: tag The adapter operation was aborted Error: The adapter operation was aborted
    at r.t [as constructor] (/tmp/broccoli-1557eQmVQFT8lvoD/out-276-uglify_writer/assets/vendor.js:2507:13)
    at r.ie (/tmp/broccoli-1557eQmVQFT8lvoD/out-276-uglify_writer/assets/vendor.js:5090:96)
    at new r (/tmp/broccoli-1557eQmVQFT8lvoD/out-276-uglify_writer/assets/vendor.js:5091:56)
    at /tmp/broccoli-1557eQmVQFT8lvoD/out-276-uglify_writer/assets/vendor.js:5538:8
    at /tmp/broccoli-1557eQmVQFT8lvoD/out-276-uglify_writer/assets/vendor.js:5538:28
    at /tmp/broccoli-1557eQmVQFT8lvoD/out-276-uglify_writer/assets/vendor.js:5539:90
    at Object.u.error (/tmp/broccoli-1557eQmVQFT8lvoD/out-276-uglify_writer/assets/vendor.js:5539:100)
    at fire (/home/salvatore/Projects/super-blog/node_modules/jquery-deferred/lib/jquery-callbacks.js:78:30)
    at Object.fireWith (/home/salvatore/Projects/super-blog/node_modules/jquery-deferred/lib/jquery-callbacks.js:188:7)
    at Object.fire [as reject] (/home/salvatore/Projects/super-blog/node_modules/jquery-deferred/lib/jquery-callbacks.js:195:10)
    at ClientRequest.onError (/home/salvatore/Projects/super-blog/node_modules/najax/lib/najax.js:208:9)
    at ClientRequest.emit (events.js:182:13)
    at Socket.socketErrorListener (_http_client.js:392:9)
    at Socket.emit (events.js:182:13)
    at emitErrorNT (internal/streams/destroy.js:82:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:50:3)
    at process._tickCallback (internal/process/next_tick.js:63:19)
pre-render /tag/getting-started failed with exception: Error: The adapter operation was aborted
cleaning up...
Build Error (prember)

Some pre-rendered URLs had failures
```

I'm on a Xubuntu 18.10, Node.js v10.15.0, NPM v6.4.1.

Any suggestion is welcome :smile: 
thanks

**Comment by moebiusmania** [comment]
it seems to happen also on MacOS 10.14.2

**Comment by mansona** [comment]
Hey @moebiusmania thanks for the bug report! 

I have investigated and it seems like the issue is related to a bug in an upstream dependency: https://github.com/ef4/prember/pull/34

If you want to get this to work straight away you can update your package json to be using `"prember": "1.0.0",` (note: no `^` and depending on exactly v1.0.0). If that works for you please let me know 👍

**Comment by moebiusmania** [comment]
hi @mansona , that seems to do the trick! 🎉🎉🎉🎉 
working on MacOS & on the Travis CI build!
thanks for the help, I'm really appreciating ember-ghost

**Comment by moebiusmania** [comment]
@mansona sure thing! it's already live here https://salvatorelaisa.blog/ , nothing fancy it's just a personal blog (in italian) 😄 .

I've started with Ghost but wanted to move on static site generator, Gatsby was a bit overkill, VuePress is still in early stages, and I've found `ember-ghost` almost randomly and sounded good for me since I was liking the Casper theme in Ghost. Easy to setup and publish content for a JS folk.

thanks for the good work!

**Comment by mansona** [comment]
That's awesome 🎉 looks pretty sweet! Also thanks for giving your reasons for trying out ember-ghost, it's sometimes hard to know why people are trying and liking something! 

Also just to let you know your original issue is now solved upstream in prember v1.0.2 so you can go back to using `^1.0.2` if you like in your package.json 👍
